Doc/library/pathlib.rst

```diff
@@ -636,6 +636,22 @@ Concrete paths provide the following methods in addition to pure paths
 methods.  Many of these methods can raise an :exc:`OSError` if a system
 call fails (for example because the path doesn't exist):
 
+.. method:: Path.absolute()
```

Comment:
Please don't document `Path.absolute()` until [bpo-29688](https://bugs.python.org/issue29688) is resolved.